images/linux/ubuntu1604.json

```diff
@@ -79,7 +79,8 @@
                 "systemctl disable apt-daily-upgrade.service",
                 "echo '* soft nofile 50000 \n* hard nofile 50000' >> /etc/security/limits.conf",
                 "echo 'session required pam_limits.so' >> /etc/pam.d/common-session",
-                "echo 'session required pam_limits.so' >> /etc/pam.d/common-session-noninteractive"
+                "echo 'session required pam_limits.so' >> /etc/pam.d/common-session-noninteractive",
+                "echo 'DefaultLimitNOFILE=50000' >> /etc/systemd/system.conf"
```

Comment:
Just wondering why we should not use Max possible number, what the motivation to set 50000?

Reply:
Previous max file limit just has been repeated in this setting

Reply:
Let's change it to the max possible value if you see not issues with it.

Reply:
Max files limit has been changed to 65536